### DISCUSSION TITLE
parcel v2 - is it possible to use url without .html at the end?

### DISCUSSION THREAD

**Discussion by jshikanova** [parcel-bundler/parcel#5379] [Q&A]
If a have multiple html pages (index.html, about.html etc) is there're any way I can refer to a page without .html at the end?

For example:
`
<a href="./about">About</a>
`
instead of 
`
<a href="./about.html">About</a>
`

**Comment by northern-mindset** [comment]
I guess by writing a namer plug-in, but I have not looked into it deeply. I am also wondering how to make parcel output more human-friendly.
